ci/test/00_setup_env_win64.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win64 (bionic is used in the gitian build as well)
+export DOCKER_NAME_TAG=ubuntu:20.04
```

Comment:
I don't like this change, but without it the ci fails: https://cirrus-ci.com/task/6121023374360576?command=ci#L5504

```
compiling .rcc/release/qrc_qmake_webgradients.cpp
In file included from rhi/qrhi.cpp:49:0:
rhi/qrhid3d11_p_p.h:57:10: fatal error: dxgi1_3.h: No such file or directory
 #include <dxgi1_3.h>
          ^~~~~~~~~~~
compilation terminated.
Makefile.Release:71663: recipe for target '.obj/release/qrhi.o' failed
make[3]: *** [.obj/release/qrhi.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from rhi/qrhid3d11.cpp:37:0:
rhi/qrhid3d11_p_p.h:57:10: fatal error: dxgi1_3.h: No such file or directory
 #include <dxgi1_3.h>
          ^~~~~~~~~~~
compilation terminated.
Makefile.Release:72684: recipe for target '.obj/release/qrhid3d11.o' failed
make[3]: *** [.obj/release/qrhid3d11.o] Error 1
make[3]: Leaving directory '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.15.2-102170435c3/qtbase/src/gui'
Makefile:45: recipe for target 'release' failed
make[2]: *** [release] Error 2
make[2]: Leaving directory '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.15.2-102170435c3/qtbase/src/gui'
Makefile:423: recipe for target 'sub-gui' failed
make[1]: *** [sub-gui] Error 2
make[1]: Leaving directory '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.15.2-102170435c3/qtbase/src'
funcs.mk:273: recipe for target '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.15.2-102170435c3/./.stamp_built' failed
make: *** [/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.15.2-102170435c3/./.stamp_built] Error 2
make: Leaving directory '/tmp/cirrus-ci-build/depends'
Exit status: 2

Reply:
Do the gitian builds pass? If yes, why do they and the ci doesn't?

Reply:
> I don't like this change, but without it the ci fails:

A newer version of mingw-w64 & headers is required to build this version of Qt, so we (essentially) have to use a newer version of Ubuntu in the CI & gitian (I'll add those changes, but will PR them separate in advance). This was also discussed above.

Reply:
Note to myself: https://github.com/bitcoin/bitcoin/pull/19716#issuecomment-767347016